pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -487,3 +563,29 @@ func (a *Operator) ensureOpGroupClusterRoles(op *v1alpha2.OperatorGroup) error {
 	}
 	return nil
 }
+
+func (a *Operator) findCSVsThatProvideAnyOf(provide resolver.APISet) ([]*v1alpha1.ClusterServiceVersion, error) {
+	csvs, err := a.lister.OperatorsV1alpha1().ClusterServiceVersionLister().ClusterServiceVersions(metav1.NamespaceAll).List(labels.Everything())
```

Comment:
If annotations are updated on the OperatorGroup, that means the shared index cache should contain any currently existing CSVs that we care about.